scripts/solhint-custom/index.js

```diff
@@ -71,14 +71,18 @@ module.exports = [
     }
   },
 
-  // TODO: re-enable and fix
-  // class extends Base {
-  //   static ruleId = 'no-external-virtual';
-  //
-  //   FunctionDefinition(node) {
-  //     if (node.visibility == 'external' && node.isVirtual) {
-  //       this.error(node, 'Functions should not be external and virtual');
-  //     }
-  //   }
-  // },
+  class extends Base {
+    static ruleId = 'no-external-virtual';
+
+    FunctionDefinition(node) {
+      // Skip constructor, function without a name (receive/fallback) and interface definitions
+      if (node.isConstructor || node.name == null || node.parent.kind === 'interface') {
```

Comment:
NOTE: prettier rules say that receive and fallback function MUST be external. This creates a conflicts if they are virtual. This leaves us tree options:

- remove virtual from receive/fallback: bit no for me!
- change the prettier config
- (what I did here) skip the rule for receive and fallback functions.